docs/docs/apis/actions/register-flow.md

```diff
@@ -105,6 +107,90 @@ A user selected **Register** on the overview page after external authentication
   The first parameter contains the following fields
   - `v1`
     - `getUser()` [*user*](./objects#user)
+    - `authRequest` [*auth request*](/docs/apis/actions/objects#auth-request)
+- `api`  
+  The second parameter contains the following fields
+  - `userGrants` Array of [*userGrant*](./objects#user-grant)'s
+  - `v1`
+    - `appendUserGrant(`[`userGrant`](./objects#user-grant)`)`
+
+## Internal Authentication
+
+### Post Authentication
+
+A user has authenticated directly at ZITADEL.
+ZITADEL validated the users inputs for password, one-time password, security key or passwordless factor.
+Each validation step triggers the action.
+
+#### Parameters of Post Authentication Action
+
+- `ctx`  
+The first parameter contains the following fields
+  - `v1`
+    - `authMethod` *string*  
+    This is one of "password", "OTP", "U2F" or "passwordless"
+    - `authError` *string*  
+    This is a verification errors string representation. If the verification succeeds, this is "none"
+    - `authRequest` [*auth request*](/docs/apis/actions/objects#auth-request)
+- `api`  
+  The second parameter contains the following fields
```

Comment:
the fields are missing
i guess they should be the same as in external authentication